lib/logs/log_formatter.rb

```diff
@@ -41,7 +41,7 @@ def call
         value = @data[key]
         key_value_string = format(key, value)
 
-        if Rails.env.local?
+        if Rails.env.development?
```

Comment:
The code within this block causes a spec to fail, so we'll execute it now only in development.